Resources/Prototypes/Entities/Clothing/OuterClothing/wintercoats.yml

```diff
@@ -430,16 +454,31 @@
   description: A sturdy, utilitarian winter coat designed to protect a warden from any brig-bound threats and hypothermic events.
   components:
   - type: Sprite
-    sprite: Clothing/OuterClothing/WinterCoats/coatwarden.rsi
+    sprite: Clothing/OuterClothing/WinterCoats/coatwardenarmored.rsi
   - type: Clothing
-    sprite: Clothing/OuterClothing/WinterCoats/coatwarden.rsi
+    sprite: Clothing/OuterClothing/WinterCoats/coatwardenarmored.rsi
   - type: Armor
     modifiers:
       coefficients:
-        Blunt: 0.70
-        Slash: 0.70
-        Piercing: 0.8 #slightly less bulletproof then warden's normal coat
-        Heat: 0.70
+        Blunt: 0.7
+        Slash: 0.7
+        Piercing: 0.7
+        Heat: 0.7
+  - type: ExplosionResistance
+    damageCoefficient: 0.9
```

Comment:
Same here, copy-pasting to 2 different values is going to be prone to bugs that are already occurring instead of fixing the root problem.